CorsixTH/Lua/entities/humanoids/staff.lua

```diff
@@ -547,20 +547,21 @@ end
 
 -- Makes the staff member request a raise of 10%, or a wage exactly in the middle of their current and a fair one, whichever is more.
 function Staff:requestRaise()
-  -- Is this the first time a member of staff is requesting a raise?
-  -- Only show the help if the player is playing the campaign though
-  if self.hospital and not self.hospital.has_seen_pay_rise and tonumber(self.world.map.level_number) then
-    self.world.ui.adviser:say(_A.information.pay_rise)
-    self.hospital.has_seen_pay_rise = true
-  end
   -- Check whether there is already a request for raise.
   if not self:isMoodActive("pay_rise") then
     local amount = math.floor(math.max(self.profile.wage * 1.1, (self.profile:getFairWage() + self.profile.wage) / 2) - self.profile.wage)
     -- At least for now, staff are timid, and only ask for raises 1/5th of the time
     if math.random(1, 5) ~= 1 or amount <= 0 then
       self.timer_until_raise = nil
-      return
+      return -- too timid
     end
+  -- They fought their timidness, time to put the raise in motion!
+  -- Is this the first time a member of staff is requesting a raise?
+  -- Only show the help if the player is playing the campaign though
+  if self.hospital and not self.hospital.has_seen_pay_rise and tonumber(self.world.map.level_number) then
+    self.world.ui.adviser:say(_A.information.pay_rise)
+    self.hospital.has_seen_pay_rise = true
+  end
```

Comment:
When moving code, adjust indentation to the new environment, the `end` at 568 is left of this `end`.